src/services/twilio.ts

```diff
@@ -45,7 +49,38 @@ export const TwilioClient = (): IPhoneProviderService => {
       return new UnknownPhoneProviderServiceError(err)
     }
 
-    logger.info({ to }, "sent text successfully")
+    return true
+  }
+
+  const validateVerify = async ({
+    to,
+    code,
+  }: {
+    to: PhoneNumber
+    code: PhoneCode
+  }): Promise<true | UnknownPhoneProviderServiceError | CodeInvalidError> => {
+    let verification: VerificationCheckInstance
+
+    try {
+      verification = await verify.verificationChecks.create({ to, code })
+    } catch (err) {
+      baseLogger.error({ err }, "impossible to verify phone and code")
+
+      if (err.message.includes("Invalid parameter `To`")) {
+        return new InvalidPhoneNumberPhoneProviderError(err)
+      }
+
+      if (err.status === 404) {
+        return new ExpiredOrNonExistentPhoneNumber(err)
```

Comment:
please use `err.message || err`

Reply:
like this: 5e23e34b42e14664bf562ad128c822924e19b304 ?